packages/gatsby-image/src/__tests__/index.js

```diff
@@ -119,6 +119,22 @@ const setupImages = (
 }
 
 describe(`<Image />`, () => {
+  const OLD_MATCH_MEDIA = window.matchMedia
```

Comment:
These tests run in nodejs which has no `window`. How are we going to test them there?

Reply:
The `devDependencies` lists `jest-environment-jsdom-fourteen`, so we have access to the window object through `js-dom` in the tests.